code/game/objects/items/weapons/cosmetics.dm

```diff
@@ -6,38 +6,49 @@
 	w_class = WEIGHT_CLASS_TINY
 	var/colour = "red"
 	var/open = 0
+	var/stickprefix = "red"
 
 /obj/item/lipstick/purple
 	name = "purple lipstick"
 	colour = "purple"
+	stickprefix = "purple"
 
 /obj/item/lipstick/jade
-	//It's still called Jade, but theres no HTML color for jade, so we use lime.
 	name = "jade lipstick"
+	colour = "#216F43"
+	stickprefix = "jade"
+
+/obj/item/lipstick/lime
+	name = "lime lipstick"
 	colour = "lime"
+	stickprefix = "lime"
 
 /obj/item/lipstick/black
 	name = "black lipstick"
 	colour = "black"
+	stickprefix = "black"
 
 /obj/item/lipstick/green
 	name = "green lipstick"
 	colour = "green"
+	stickprefix = "green"
 
 /obj/item/lipstick/blue
 	name = "blue lipstick"
 	colour = "blue"
+	stickprefix = "blue"
 
 /obj/item/lipstick/white
 	name = "white lipstick"
 	colour = "white"
+	stickprefix = "white"
 
 /obj/item/lipstick/random
 	name = "lipstick"
 
 /obj/item/lipstick/random/New()
-	colour = pick("red","purple","lime","black","green","blue","white")
-	name = "[colour] lipstick"
+	colour = pick("red","purple","lime","black","green","blue","white","#216F43")
+	name = "[stickprefix] lipstick"

```

Comment:
This change won't work. `stickprefix` is never assigned to again, here, so it'll just inherit "red". Instead, you might want to do something such as:
```
var/list/lipstick_colors = list(
    "red" = "red",
    "purple" = "purple",
...
    "jade" = "#216F43"
)
    var/lscolor = pick(lipstick_colors) // this will choose a key in the list, one of the "left-hand" values
    colour = lipstick_colors[lscolor] // this will get the "right-hand" color for that key
    name = "[lscolor] lipstick"
```
With this code, you should also be able to remove the `stickprefix` variable you currently implement.